lib/calendar_translations/translations.ex

```diff
@@ -535,6 +535,112 @@ defmodule CalendarTranslations.Translations do
   def date_format(:"zh-TW"), do: {:ok, "%Y-%m-%d"}
   def date_format(:"zh-YUE"), do: {:ok, "%Y-%m-%d"}
 
+  def time_format(:af), do: {:ok, "%H:%M"}
+  def time_format(:ar), do: {:ok, "%H:%M"}
+  def time_format(:az), do: {:ok, "%H:%M"}
+  def time_format(:be), do: {:ok, "%H:%M"}
+  def time_format(:bg), do: {:ok, "%H:%M"}
+  def time_format(:bn), do: {:ok, "%H:%M"}
+  def time_format(:bs), do: {:ok, "%H:%M:%S"}
+  def time_format(:ca), do: {:ok, "%H:%M"}
+  def time_format(:cs), do: {:ok, "%H:%M"}
+  def time_format(:csb), do: {:ok, "%H:%M"}
+  def time_format(:cy), do: {:ok, "%H:%M"}
+  def time_format(:da), do: {:ok, "%H.%M"}
+  def time_format(:de), do: {:ok, "%H:%M Uhr"}
+  def time_format(:"de-AT"), do: {:ok, "%H:%M Uhr"}
+  def time_format(:"de-CH"), do: {:ok, "%H:%M Uhr"}
+  def time_format(:dsb), do: {:ok, "%H:%M góź."}
+  def time_format(:el), do: {:ok, "%H:%M:%S"}
+  def time_format(:en), do: {:ok, "%H:%M"}
+  def time_format(:"en-AU"), do: {:ok, "%H:%M"}
+  def time_format(:"en-CA"), do: {:ok, "%I:%M %p"}
+  def time_format(:"en-GB"), do: {:ok, "%H:%M"}
+  def time_format(:"en-IE"), do: {:ok, "%H:%M"}
+  def time_format(:"en-IN"), do: {:ok, "%H:%M"}
+  def time_format(:"en-NZ"), do: {:ok, "%H:%M"}
+  def time_format(:"en-US"), do: {:ok, "%I:%M %p"}
+  def time_format(:"en-ZA"), do: {:ok, "%H:%M"}
+  def time_format(:eo), do: {:ok, "%H:%M"}
+  def time_format(:es), do: {:ok, "%H:%M"}
+  def time_format(:"es-419"), do: {:ok, "%I:%M %p"}
+  def time_format(:"es-AR"), do: {:ok, "%I:%M %p"}
+  def time_format(:"es-CL"), do: {:ok, "%H:%M"}
+  def time_format(:"es-CO"), do: {:ok, "%I:%M %p"}
+  def time_format(:"es-CR"), do: {:ok, "%I:%M %p"}
+  def time_format(:"es-EC"), do: {:ok, "%-I:%M %p"}
+  def time_format(:"es-MX"), do: {:ok, "%I:%M %p"}
+  def time_format(:"es-PA"), do: {:ok, "%-I:%M %p"}
```

Comment:
What is `%-I`? Can't seem to find it [in the docs](https://hexdocs.pm/calendar/Calendar.Strftime.html#strftime/3).